cp1616/config/test_io_controller_config.yaml

```diff
@@ -0,0 +1,43 @@
+#Data comming from PROFINET network
+input:
```

Comment:
The same situation as with launch files, forgot to add new one, this worked with previous  XMLRpc parsing, uploaded to simple flat data structutre